src/wp-includes/webfonts.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * Webfonts API: Webfonts functions
+ *
+ * @since 5.9.0
+ *
+ * @package WordPress
+ * @subpackage Webfonts
+ */
+
+/**
+ * Instantiates the webfonts controller, if not already set, and returns it.
+ *
+ * @since 5.9.0
+ *
+ * @return WP_Webfonts_Controller Instance of the controller.
+ */
+function wp_webfonts() {
+	static $instance;
+
+	if ( ! $instance instanceof WP_Webfonts ) {
+		$instance = new WP_Webfonts_Controller(
+			new WP_Webfonts_Registry(
+				new WP_Webfonts_Schema_Validator()
+			),
+			new WP_Webfonts_Provider_Registry()
+		);
+		$instance->init();
+	}
+
+	return $instance;
+}
+
+/**
+ * Registers a webfont collection.
+ *
+ * @since 5.9.0
+ *
+ * @param array $webfonts Webfonts to be registered.
+ *                        This contains ar array of webfonts to be registered. Each webfont is an array.
```

Comment:
```suggestion
 *                        This contains an array of webfonts to be registered. Each webfont is an array.
```

Minor typo

Reply:
Thank you @costdev - fixed in https://github.com/WordPress/wordpress-develop/pull/1736/commits/92bd91195f7896404e666f8a091b7224903157e9.